### PR TITLE
Devengage 1529 add member groups to bullseye

### PR DESCRIPTION
**Pull request by carnellj-genesys**

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -0,0 +1,720 @@
+go test --tags=flows ./... -v  -timeout 120m -parallel 20
```
Comment (charliecon) [inline test.txt:1]: Accidentally checked in file

File: genesyscloud/resource_genesyscloud_tf_export.go [hunk 2]
```diff
@@ -1183,6 +1197,7 @@ func resolveRefAttributesInJsonString(currAttr string, currVal string, exporter
 	return string(jsonDataMarshalled), nil
 }
 
+// Could we just do an OK on the map without having to
```
Comment (charliecon) [inline genesyscloud/resource_genesyscloud_tf_export.go:1200]: Leftover brainstorming comment

File: examples/resources/genesyscloud_routing_queue/resource.tf [hunk 3]
```diff
@@ -29,6 +29,11 @@ resource "genesyscloud_routing_queue" "example_queue" {
   bullseye_rings {
     expansion_timeout_seconds = 15.1
     skills_to_remove          = [genesyscloud_routing_skill.example-skill.id]
+
+    member_groups = {
```
Comment (charliecon) [inline examples/resources/genesyscloud_routing_queue/resource.tf:33]: `member_groups = {`  -->  `member_groups {`

File: genesyscloud/resource_exporter_custom.go [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+*/
+func MemberGroupsResolver(configMap map[string]interface{}, exporters map[string]*ResourceExporter) error {
+
+	var exporter *ResourceExporter
+	memberGroupType := configMap["member_group_type"]
+	memberGroupID := configMap["member_group_id"].(string)
+
+	switch memberGroupType {
+	case "SKILLGROUP":
+		exporter = exporters["genesyscloud_routing_skill_group"]
```
Comment (charliecon) [inline genesyscloud/resource_exporter_custom.go:23]: There needs to be some check here to see if this key exists in the map. If a user forgets to include `genesyscloud_routing_skill_group` or `genesyscloud_group` in the `resource_types` array when they're exporting, they'll see a panic nil pointer exception

File: test.txt (hunk 1 above)
Comment (carnellj-genesys) [reply test.txt:1]: Fixed

File: genesyscloud/resource_genesyscloud_tf_export.go (hunk 2 above)
Comment (carnellj-genesys) [reply genesyscloud/resource_genesyscloud_tf_export.go:1200]: Done

**Review by charliecon — APPROVED** [review]

**Pull request merged**